Mu2eG4/geom/muonBeamStop_v08.txt

```diff
@@ -0,0 +1,13 @@
+// Muon Beam Dump/Stop
+
+#include "Mu2eG4/geom/muonBeamStop_v07.txt"
+
+//turn on absorber for STM
```

Comment:
Shouldn't mbs.Version be changed?

Reply:
I didn't increment the version number, as I typically do this when there's a significant enough change that the version needs to be used inside the code. I can increment the number here though and make sure there are no `version == 7` statements in the code though to ensure the version number matches the file version.

Reply:
I've updated the version numbers and one check in the code for the Saddles and MBS